checks/proxy.rb

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env ruby
+
+begin
+  require 'rubygems'
+rescue LoadError
+end
+
+# checking to see if http/https proxy envs are set before running the installer
+
+http_proxy = ENV['http_proxy']
+https_proxy = ENV['https_proxy']
+
+PROXYSET = "Currently there is a environmental proxy set to reach out to the internet, Katello does a check via wget to make sure services are running correctly. Please unset this and then run katello-installer again."
```

Comment:
Nitpick, but this shouldn't be a constant (all uppercase creates a constant), it'll never be garbage collected